db/queries/choices.js

```diff
@@ -22,6 +32,20 @@ const getChoices = function(pollId) {
 
 const getChoicesandscore = function(pollId) {
   return db.query(`
+<<<<<<< HEAD
+  SELECT choices.id,value, sum(6-votes.ranking) as score FROM choices
+  join votes on choices.id = votes.choice_id
+  where choices.poll_id = $1 GROUP BY choices.id,value order by choices.id ;
+  `,[pollId])
+
+  .then((result) => {
+
+    return result.rows;
+  })
+  .catch((err) => {
+    console.log(err.message);
+  });
+=======
```

Comment:
this function is fixed, please keep the master version of it.

Reply:
again, please use the master version of this function, as it was refactored to fix the zero votes bug